06.bowling_object/game.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Game
+  LAST_FRAME = 9
+
+  def initialize(input_marks)
+    @frames = create_frames(input_marks)
+  end
+
+  def score
+    @frames.sum do |frame|
+      frame.score(@frames)
+    end
+  end
+
+  private
+
+  def create_frames(input_marks)
+    frame_shots = []
+    index = 0
+
+    converted_shots = input_marks.split(',').map { |mark| Shot.new(mark) }
```

Comment:
```suggestion
    all_shots = input_marks.split(',').map { |mark| Shot.new(mark) }
```

なんとなく、ゲーム内の全shotが格納された配列、と表現する方がわかりやすい気がしました
（変換されてできたかどうかは読み手にとってあまり興味がない）

Reply:
修正しました。